### DISCUSSION TITLE
Random moments of black screen during gameplay

### DISCUSSION THREAD

**Discussion by BenCheung0422** [ppy/osu#27611] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Sometimes during gameplay, I can experience some moments of black screen, but I cannot spot out what causes the issue, nothing special found in both game logs and system logs. Fortunately, I can capture the moment when I was recording, and interestingly, black screen moment also occurs when I play the recording in full-screen again. Therefore, I think it might be related to some loss of frames. It occurred sometimes (but rarely) when I just started playing a map.

### Screenshots or videos

Please note that playing this on GitHub does not reflect the incident of black screen for me, even in full-screen.

https://github.com/ppy/osu/assets/74168521/856cfdb6-ff6f-4493-af3f-9b80ec60b877

### Version

2024.312.1

### Logs

[logs-1710415718.zip](https://github.com/ppy/osu/files/14603607/logs-1710415718.zip)

**Comment by bdach** [comment]
I don't see any "black screens" on the video though?

**Comment by BenCheung0422** [comment]
When I play it in full-screen locally, my monitor does respond that.

**Comment by bdach** [comment]
Wait. Are you saying that your _screen_ turns black even when you're watching the _video_ of this?

I'm sorry but that sounds like you may be describing a hardware issue.

**Comment by bdach** [comment]
Played the video back in 0.12x in VLC and I see no black frames.

Converting to discussion but I'm generally confident this is probably going to be your monitor or something.

**Comment by BenCheung0422** [comment]
How can that be related to specifically a monitor? My monitor does turn black for a moment when I play the video in Windows' video playing apps.

**Reply by bdach** [reply]
How else do you explain the fact that I don't see a single frame of black on the video you linked on at least 2 other devices? Unless it's github transcoding the video or something, but if that is the case, then I'm not sure there's any usable evidence to look at in here.

**Reply by BenCheung0422** [reply]
I am not sure what causes the black screen, but it does indeed happen on my monitor. Probably it occurs only on some particular configuration or devices.

**Reply by Gabixel** [reply]
A somewhat random guess is that it's not configured well for certain colours (but I don't know much about this, so I might be talking nonsense, but that's what I've noticed from my experience). I have a secondary monitor that, when it tries to display certain colour ranges (whether it's on lazer or other apps, doesn't matter), also goes dark. (And it's a bit ancient...)

**Reply by BlockHead007** [reply]
What video output type are you using? I had same issue with my old hdmi to vga converter, sometimes monitor was losing signal for few seconds, so whole screen was going black. Then I bought a new one and it looks like the problem is gone. If you use something like this then try to use different connection type, try use different monitor or a different cable, it should be something from the list.

**Reply by BenCheung0422** [reply, marked as answer]
@BlockHead007 It is HTMI (GPU) to VGA (monitor) with an adapter. It seems like it is the similar situation to yours. When I removed the adapter and directly plugged it to the HTMI port (occupied by another device) of the monitor. The beat map that always causes black screen at the beginning no longer causes black screen. Then, I have searched about the reasons, including power supply, interface, cable and driver, but it is a rare case that this problem only happens at particular some of the moments in osu! from my experience. Anyway, it is possible that it is related to some signals not handled correctly by the HTMI-to-VGA adapter.
